machine_learning_hep/data/data_prod_20200304/database_ml_parameters_LcpK0spp_0304_HM_V0.yml

```diff
@@ -370,8 +370,6 @@ LcpK0spp:
         data: "trigger_hasbit_HighMultV0==1"
         mc: null
         usetriggcorrfunc: Null
-      get_crossmb_from_path: True
-      path_for_crossmb: /data/DerivedResults/LckAnywithJets_sub/vAN-20200304_ROOT6-1/pp_data/resultsMBvspt_perc_v0m/finalcrossLcpK0sppMBvspt_perc_v0mmult3.root
```

Comment:
You should keep both these variables in your database. They are used to get the fprompt from MB, so should point to path_to_MB/finalcrossLcpK0sppMBvspt_perc_v0mmult0. Now you will calculate the fprompt fraction with just the HM ingredients, but the feeddown subtraction method (Nb) doesn't work with a multiplicity selection.

Reply:
Yeah, that is a good question why I deleted that...